test/suites/integration/scheduling_test.go

```diff
@@ -416,7 +416,7 @@ var _ = Describe("Scheduling", Ordered, ContinueOnFailure, func() {
 		env.ExpectCreated(pod, nodeClass, nodePoolLowPri, nodePoolHighPri)
 		env.EventuallyExpectHealthy(pod)
 		env.ExpectCreatedNodeCount("==", 1)
-		Expect(ptr.StringValue(env.GetInstance(pod.Spec.NodeName).InstanceType)).To(Equal("c4.large"))
+		Expect(ptr.StringValue(env.GetInstance(pod.Spec.NodeName).InstanceType)).To(Equal("c5.large"))
```

Comment:
Why are these changes worked in here? How do these apply to the PR?

Reply:
`eu-north-1` does not contain these instance types.